### DISCUSSION TITLE
Clarification on upgrade needed v0.18.5 --> latest

### DISCUSSION THREAD

**Discussion by var-log-skynet** [gravitl/netmaker#2389] [Q&A]
Hi, 
documentation v0.20.1 has a paragraph in the upgrades section: "Upgrading server and client to 0.18.5" (https://docs.netmaker.org/upgrades.html#upgrading-server-and-client-to-0-18-5), but nothing from there to latest version. Could someone please describe the upgrade process from 0.18.5 to 0.20.1?

**Comment by miyu4u** [comment]
I gave up and did a clean install. The configuration structure of docker-compose has changed since v0.20.0, and the latest version uses so many subdomains that it's a waste of time to use diff to analyze what's changed and upgrade. If you only use a small number of machines, it's better to just do a clean install.
